forward/fwd.go

```diff
@@ -190,18 +210,6 @@ func (f *httpForwarder) serveHTTP(w http.ResponseWriter, req *http.Request, ctx
 		ctx.log.Infof("Round trip: %v, code: %v, duration: %v",
 			req.URL, response.StatusCode, time.Now().UTC().Sub(start))
 	}
-
-	defer response.Body.Close()
-
-	if err != nil {
-		ctx.log.Errorf("Error copying upstream response Body: %v", err)
-		ctx.errHandler.ServeHTTP(w, req, err)
-		return
-	}
-
-	if written != 0 {
```

Comment:
Okay, in testing, didn't know. For the sake of completeness, is this forbidden by Go (and possibly documented somewhere) or rather an underlying HTTP requirement?